src/order/complete_boolean_algebra.lean

```diff
@@ -1,36 +1,97 @@
 /-
 Copyright (c) 2017 Johannes Hölzl. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Johannes Hölzl
+Authors: Johannes Hölzl, Yaël Dillies
 -/
 import order.complete_lattice
 
 /-!
-# Completely distributive lattices and Boolean algebras
+# Frames, completely distributive lattices and Boolean algebras
 
-In this file there are definitions and an API for completely distributive lattices and completely
+In this file we define and provide API for frames, completely distributive lattices and completely
 distributive Boolean algebras.
 
 ## Typeclasses
 
+* `frame`: Frame: A complete lattice whose `⊓` distributes over `⨆`.
 * `complete_distrib_lattice`: Completely distributive lattices: A complete lattice whose `⊓` and `⊔`
   distribute over `⨆` and `⨅` respectively.
 * `complete_boolean_algebra`: Completely distributive Boolean algebra: A Boolean algebra whose `⊓`
   and `⊔` distribute over `⨆` and `⨅` respectively.
```

Comment:
Could you please expand the docstring with examples of useful types that belong to one of these typeclasses but not stronger typeclasses.

Reply:
Otherwise LGTM.

Reply:
Hmm... I don't know any, so feel free to suggest.